qa/remote-clusters/docker-compose.yml

```diff
@@ -67,8 +67,19 @@ services:
       timeout: 2s
       retries: 5
   haproxy:
+    build:
+      context: .
+      dockerfile: Dockerfile
     image: haproxy:2.1.2
     ports:
       - "9600"
     volumes:
       - ./haproxy.cfg:/usr/local/etc/haproxy/haproxy.cfg:ro
+    depends_on:
+      - opensearch-2
+    healthcheck:
+      test: ["CMD", "nc", "-z", "localhost", "9600"]
```

Comment:
Can we use something like ```curl http://localhost:9600/healthcheck``` for doing the haproxy healthcheck?

Reply:
We cannot do that because 9600 is a tcp port, it's a proxy of the 9300 port in OpenSearch.

Reply:
Thanks @gaobinlong , it seems like we could just `bash` (since `nc` is not installed by default):

> timeout 1 bash -c 'cat < /dev/null > /dev/tcp/localhost/9600'

WDYT?

Reply:
This solution looks good to me, I tried that but failed because I used `sh`, not `bash`, I've changed it to this method, thank you!